appveyor.yml

```diff
@@ -5,15 +5,17 @@ environment:
 install:
   - ps: Install-Product node $env:nodejs_version x64
   - node --version
-  - yarn --version
-  - yarn install
+  - curl -fsSL -o yarn.js https://github.com/yarnpkg/yarn/releases/download/v0.27.5/yarn-0.27.5.js
```

Comment:
To me it is just more standard to rely on a global yarn installation. I hope appveyour global Yarn installation will be overridden by a `npm install -g`.